### DISCUSSION TITLE
Hacktoberhfest 2023

### DISCUSSION THREAD

**Discussion by ayo-run** [ayoayco/astro-reactive-library#282] [Ideas]
This coming October, we anticipate another great hackathon with Hacktoberfest.

We will participate, of course! :)

My idea for this year is not to open the library, but a new separate project app that uses our library. 

I'm thinking of a dynamic form builder that will enable users to create surveys/quizzes visually. 

Sounds good? Let me know if you have ideas! :)

**Comment by Lalit3716** [comment]
Why not open both the library as well as project app for hacktoberfest 🙂!

**Comment by fazzaamiarso** [comment]
Since we are creating a new playground, maybe we can let contributors create more examples using our library.

**Comment by Preet-Sojitra** [comment]
I agree with Lalit and fazzaamiarso

**Comment by ayo-run** [comment]
Hey guys @Lalit3716 @fazzaamiarso @Preet-Sojitra, I like the ideas! It's just that during hacktoberfest there is really going to be a lot to review 😆 So I was wondering if we can divide the tasks... if you want to participate as a Maintainer this year and you have access to this organization, we can all review PRs. :)

I have started a new project: http://github.com/ayoayco/cozy-reader -- it's a fun one, that I think will help a lot of people. Also, it uses our Form component here in a very simple use case: the Address Bar. Check it out! I plan to open that one for hacktoberfest too so if you want to participate as a Contributor, that's also a good use case

**Reply by Lalit3716** [reply]
Sure I will love to be a maintainer this Hacktoberfest! Yeah new repository looks good it may attract many contributors 😄👍
